File: src/test/java/kitchenpos/apply/menu/tobe/infra/FakePurgomalumClient.java
package kitchenpos.apply.menu.tobe.infra;


import kitchenpos.apply.menus.tobe.domain.PurgomalumClient;

import java.util.Arrays;
import java.util.List;

public class FakePurgomalumClient implements PurgomalumClient {
    private static final List<String> profanities;

    static {
        profanities = Arrays.asList("비속어", "욕설");
    }

    @Override
    public boolean containsProfanity(final String text) {
        return profanities.stream()
            .anyMatch(text::contains);
    }
}

File: src/test/java/kitchenpos/menus/tobe/infra/FakePurgomalumClient.kt
package kitchenpos.menus.tobe.infra

class FakePurgomalumClient(vararg word: String) : PurgomalumClient {
    private val profanities: List<String> = word.toList()

    override fun containsProfanity(text: String): Boolean {
        return profanities.stream()
            .anyMatch { profanity: String? -> text.contains(profanity!!) }
    }
}
